### ISSUE TITLE
Input type=range with "multiple" attribute

### ISSUE THREAD

**Issue by valioDOTch** [Fyrd/caniuse#1804]
Currently only the standard input type=range is documented [1].

As per W3C specification [2] the range input type can have a "multiple" attribute, which should show multiple sliders on the range input.

I suggest to add a separate evaluation as with input type=file which can have a "multiple" attribute as well and has been documented [3].

I personally would prefer to have this feature in major browsers and thus consider this to be of a higher priority.

---

[1] http://caniuse.com/#feat=input-range
[2] https://html.spec.whatwg.org/multipage/forms.html#range-state-(type=range)
[3] http://caniuse.com/#feat=input-file-multiple

**Comment by nilssolanki** [comment]
Yes, this would be very useful, as currently no browser seems to support it.

**Comment by Ugoku** [comment]
+1
Polyfill by @LeaVerou here: https://leaverou.github.io/multirange/
